apps/src/lib/ui/accounts/ChangeEmailModal.jsx

```diff
@@ -2,10 +2,9 @@ import React, {PropTypes} from 'react';
 import BaseDialog from '@cdo/apps/templates/BaseDialog';
 import i18n from '@cdo/locale';
 import color from '@cdo/apps/util/color';
-import {hashEmail} from '../../code-studio/hashEmail';
-import {isEmail} from '../../util/formatValidation';
-import $ from 'jquery';
-import {Header, ConfirmCancelFooter} from './SystemDialog/SystemDialog';
+import {hashEmail} from '../../../code-studio/hashEmail';
```

Comment:
If we're using `@cdo` above, does it make sense to use it here too?

Reply:
That's a mistake on my part, I actually don't want to use the `@cdo/apps` alias in this file.  Thanks!

Reply:
Out of curiosity, why not?

Reply:
Mostly editor support: RubyMine follows regular/relative imports to provide navigation to function definitions, hints about unused exports, etc. but the `@cdo/apps` alias is a setting in our Webpack config, which is currently unparseable by the editor (I think it's just too dynamic/complicated).

On a related theme, `@cdo/apps` is generally going to have less support from various tools since it's not a standard JS feature, but a Webpack-specific one.

My habit lately is to do the following, but I'm open to changing this if there's a different practice we'd like to standardize on:

- For most code in `apps/src` I keep imports relative unless it's some other special-purpose alias, like our `@cdo/locale` imports.
- Code in `apps/src/sites` (entry points) use `@cdo/apps` absolute references to refer to other source code (this seems to be our convention).
- Test code uses relative imports to refer to other test code, but `@cdo/apps` absolute references to source code.

Reply:
Makes sense, I'm all for increased RubyMine (and additional tool) support :)